packages/gcf-utils/src/gcf-utils.ts

```diff
@@ -1310,7 +1313,15 @@ function parseRateLimitError(e: Error): RateLimits | undefined {
  * @param {GCFLogger} logger The logger to log to
  * @param {Error} e The error to log
  */
-function logErrors(logger: GCFLogger, e: Error) {
+export function logErrors(logger: GCFLogger, e: Error) {
+  // Add "@type" bindings so that Cloud Error Reporting will capture these logs.
+  const bindings = logger.getBindings;
+  if (bindings['@type'] !== ERROR_REPORTING_TYPE_NAME) {
+    logger = logger.child({
+      '@type': ERROR_REPORTING_TYPE_NAME,
+      ...bindings,
```

Comment:
Hmm, for some reason I thought it would should have maintained the existing bindings for you and merged in new entries, but reading the source, it doesn't appear that way.

Reply:
Yeah I think logger.child has a bug (forgetting to merge parent's bindings)

Reply:
Filed #4439